update-tests/src/test/java/io/zeebe/test/ContainerState.java

```diff
@@ -92,6 +97,9 @@ public void start(final boolean enableDebug) {
     broker =
         new ZeebeContainer("camunda/zeebe:" + brokerVersion)
             .withEnv("ZEEBE_LOG_LEVEL", "DEBUG")
+            .withEnv("ZEEBE_BROKER_NETWORK_MAXMESSAGESIZE", "128KB")
+            .withEnv("ZEEBE_BROKER_DATA_USEMMAP", "true")
+            .withEnv("ZEEBE_BROKER_DATA_LOGSEGMENTSIZE", "64MB")
```

Comment:
These settings are known to improve performance - however they're not defaults as changing them would be non-backwards compatible. We can remove them here once that's the case.